libafl/src/statistics/manager/global_stats.rs

```diff
@@ -0,0 +1,173 @@
+//! Global statistics available for Monitors to use
+
+use alloc::string::{String, ToString};
+use core::{cmp, time::Duration};
+
+use libafl_bolts::{current_time, format_duration_hms};
+#[cfg(feature = "std")]
+use serde_json::Value;
+
+use super::ClientStatsManager;
+use crate::statistics::ProcessTiming;
+#[cfg(feature = "std")]
+use crate::statistics::{
+    user_stats::{AggregatorOps, UserStats, UserStatsValue},
+    ItemGeometry,
+};
+
+impl ClientStatsManager {
+    /// Time this fuzzing run stated
+    #[must_use]
+    pub fn start_time(&self) -> Duration {
+        self.start_time
+    }
+
+    /// Time this fuzzing run stated
+    pub fn set_start_time(&mut self, time: Duration) {
+        self.start_time = time;
+    }
+
+    /// Get global stats.
+    ///
+    /// This global stats will be cached until the underlined client stats are modified.
+    pub fn global_stats(&mut self) -> &GlobalStats {
+        let global_stats = self.cached_global_stats.get_or_insert_with(|| GlobalStats {
```

Comment:
Might make sense to implement `default` for this struct, and then just call that here

Reply:
Yes you are right. Will push a commit sooner.